### DISCUSSION TITLE
Groups/Folders for entries

### DISCUSSION THREAD

**Discussion by mlippert** [helloworld1/FreeOTPPlus#252] [Ideas]
FreeOTPPlus is great, but as I get more and more entries (which is a great thing because more and more sites are adding 2FA TOTP support), it is getting harder to locate the entry I need, or at least it involves a lot of scrolling!

I think it would help if I could group the entries into folders that can be opened and closed, I'm not even suggested being able to nest folders, just a single level.

For example:

Stores/
Social Media/
Financial/
Games/
Healthcare/
Government/
Development/
Work/

**Comment by helloworld1** [comment]
I am thinking about adding categories:
https://github.com/helloworld1/FreeOTPPlus/issues/85
Would it cover your case?

**Comment by mlippert** [comment]
I think it would, I think it might even be better particularly if an entry could belong to multiple categories. Even if an entry can only belong to a single category, I think that is pretty much exactly what I was suggesting in this feature request.
Thanks.

**Comment by helloworld1** [comment]
Thanks. Closing this issue and discuss in the #85
